static/js/ajax-notifications.js

```diff
@@ -39,7 +42,7 @@ function wsCheckNotifications() {
 
         // Add handling for different kinds of events. For ex: {"event": "notification", "data": {"title": ..}}
         if (msg.event === 'notification') {
-            displayPNotify(msg.data.type, msg.data.title, msg.data.body);
+            displayPNotify(msg.data.type, msg.data.title, msg.data.body, msg.data.hash);
         } else {
             displayPNotify('info', '', msg);
```

Comment:
What does the response from /ui/get_messages give you? (Open the network tab, then generate a notification from Medusa)